pysb/simulator/scipyode.py

```diff
@@ -592,3 +638,65 @@ def process(self, msg, kwargs):
     warn = logging.LoggerAdapter.info
     # Provide 'fatal' to match up with distutils log functions.
     fatal = logging.LoggerAdapter.critical
+
+def _lsoda_process(code_eqs, jac_eqs, num_species, num_odes, *args, **kwargs):
```

Comment:
Thanks, I wasn't familar with the `concurrent.futures` approach. I've implemented it as you suggested, and added a serial executor to avoid generating an extraneous process for the single processor version.